app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
```

Comment:
Import Ordering:
It's a good practice to order imports in a certain way. Typically, you'd start with standard library imports, then third-party library imports, and finally your project's local imports.